From e8fa11ce31a64710421b67bb97d28455cdedc841 Mon Sep 17 00:00:00 2001
From: Peter Vicman <peter.vicman@gmail.com>
Date: Sun, 9 Oct 2016 14:03:21 +0200
Subject: [PATCH] remove debug ouput

disable dmesg flooding
[ 1351.747814] RTL871X: set group key camid:6, addr:fc:f5:13:a7:d5:a9, kid:1, type:AES
[ 1351.747814] RTL871X: send eapol packet

---
 core/rtw_mlme_ext.c | 4 ++--
 core/rtw_xmit.c     | 2 +-
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/core/rtw_mlme_ext.c b/core/rtw_mlme_ext.c
index 9b7a7ef..1bb065a 100644
--- a/core/rtw_mlme_ext.c
+++ b/core/rtw_mlme_ext.c
@@ -14330,8 +14330,8 @@ u8 setkey_hdl(_adapter *padapter, u8 *pbuf)
     #endif
 
     ctrl = BIT(15) | BIT6 | ((pparm->algorithm) << 2) | pparm->keyid;
-    DBG_871X_LEVEL(_drv_always_, "set group key camid:%d, addr:"MAC_FMT", kid:%d, type:%s\n"
-        , cam_id, MAC_ARG(addr), pparm->keyid, security_type_str(pparm->algorithm));
+    //DBG_871X_LEVEL(_drv_always_, "set group key camid:%d, addr:"MAC_FMT", kid:%d, type:%s\n"
+    //    , cam_id, MAC_ARG(addr), pparm->keyid, security_type_str(pparm->algorithm));
     write_cam(padapter, cam_id, ctrl, addr, pparm->key);
 
     #ifdef DYNAMIC_CAMID_ALLOC
diff --git a/core/rtw_xmit.c b/core/rtw_xmit.c
index 7b384e5..933d816 100644
--- a/core/rtw_xmit.c
+++ b/core/rtw_xmit.c
@@ -1104,7 +1104,7 @@ static s32 update_attrib(_adapter *padapter, _pkt *pkt, struct pkt_attrib *pattr
 
         
     } else if (0x888e == pattrib->ether_type) {
-        DBG_871X_LEVEL(_drv_always_, "send eapol packet\n");
+        // DBG_871X_LEVEL(_drv_always_, "send eapol packet\n");
     }
 
     if ( (pattrib->ether_type == 0x888e) || (pattrib->dhcp_pkt == 1) )
-- 
2.7.1

